docs/static/running-logstash-kubernetes.asciidoc

```diff
@@ -0,0 +1,6 @@
+[[running-logstash-kubernetes]]
+=== Running Logstash on Kubernetes
+{ls} runs on a Kubernetes cluster through Elastic Cloud on Kubernetes (ECK) in technical preview.
```

Comment:
```suggestion
{ls} runs on a {k8s} cluster through https://www.elastic.co/guide/en/cloud-on-k8s/current/index.html[{eck} (ECK)] in technical preview.
```

Reply:
We should introduce and link the Elastic Cloud on Kubernetes docs at the beginning.